hawkbit-rest/hawkbit-ddi-api/src/main/java/org/eclipse/hawkbit/ddi/rest/api/DdiRootControllerRestApi.java

```diff
@@ -373,7 +373,7 @@ ResponseEntity<Void> postDeploymentBaseActionFeedback(@Valid final DdiActionFeed
         information that will allow the server to identify the device on a hardware level (e.g. hardware revision,
         mac address, serial number etc.).""")
     @ApiResponses(value = {
-        @ApiResponse(responseCode = "200", description = "Successfully retrieved"),
+        @ApiResponse(responseCode = "201", description = "Successfully updated"),
```

Comment:
this PUT method does not create a new content - it just updates current configuration data. So I would stick with 200 here.